types/auth.ts

```diff
@@ -20,3 +20,9 @@ export interface jwtAccessTokenResponse {
   grantType: string;
   refreshToken: string;
 }
+
+export enum LoginUserState {
+  NO_USER = 'noUser',
+  WRONG_PLATFORM = 'wrongPlatform',
+  INVITE_MISMATCH = 'inviteMismatch',
+}
```

Comment:
혹여나 제가 모르는 typescript에서의 업데이트를 하지 않았다면,,,
https://engineering.linecorp.com/ko/blog/typescript-enum-tree-shaking
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html
읽어보시면 좋을 것 같습니다...!

```suggestion
export const LoginUserState = {
  NO_USER = 'noUser',
  WRONG_PLATFORM = 'wrongPlatform',
  INVITE_MISMATCH = 'inviteMismatch',
} as const
```
tree shaking 이슈를 위해서,,,`as const`로 부탁드립니당
만약 문제가 없는 부분이었던거라면,,제가 죄송합니다 :)